Added white_list supported values and usage details (#3987)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/global.rst

white_list
^^^^^^^^^^

This specifies an IPv4 address, netblock, or hostname for which Active Responses will not be triggered. Only one of those
values may be specified for each ``<while_list>`` tag, but several values may be used by including multiple
``<white_list>`` tags. This configuration is compared against the extracted **srcip** field in the alert.

+--------------------+---------------------------------------------------------------+
| **Default value**  | n/a                                                           |
+--------------------+---------------------------------------------------------------+
| **Allowed values** | Any IPv4 address, netblock (i.e.: 192.168.0.0/16) or hostname |
+--------------------+---------------------------------------------------------------+

.. note::

>   This option is only valid in server and local installs. Also, IPv4-mapped IPv6 addresses are allowed too for
>   comparation.

Reviewer: ```suggestion
  This option is only valid in server and local installs.
  IPv4-mapped IPv6 addresses will be compared against any configured IPv4 or netblock.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/395a413280b2ca7ed8a4531e6000a73c1879e6d3)

white_list
^^^^^^^^^^

> This specifies an IP, netblock, or hostname for which Active Responses will not be triggered. Only one of those values may

Reviewer: ```suggestion
This specifies an IPv4 Address, netblock, IPv4-mapped IPv6 address or hostname for which Active Responses will not be triggered. Only one of those values may
```

Author: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

be specified for each ``<while_list>`` tag, but several values may be used by including multiple ``<white_list>`` tags.

> For this to work the field **srcip** needs to be extracted from the event in the correspondent decoder as it is the one that
> will be use to compare against the values defined in white_list.

Reviewer: ```suggestion
This configuration is compared against the extracted **srcip** field in the alert.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

+--------------------+-----------------------------------------------------------+
| **Default value**  | n/a                                                       |
+--------------------+-----------------------------------------------------------+

> | **Allowed values** | Any IP address, netblock (Ej: 192.168.0.0/16) or hostname |

Reviewer: ```suggestion
| **Allowed values** | Any IPv4 address, IPv4-mapped IPv6 address, netblock (i.e.: 192.168.0.0/16) or hostname |
```

Author: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

+--------------------+-----------------------------------------------------------+

.. note::

>   This option is only valid in server and local installs. Also, wazuh supports IPv4 mapped into IPv6 if the
>   IPv6 address contains the prefix **::ffff:**, e.g, from ::ffff:192.168.1.17 the ip 192.168.1.17 would be extracted.

Reviewer: ```suggestion
  This option is only valid in server and local installs.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)
